fix: clarify "Replace" in I-D submission form

ghwood opened the pull request:
Update help text In the "Replacement Information" section of the "Status" tab to fix #8059

ghwood committed:
Clarify "Replace" in I-D submission form

Update help text In the "Replacement Information" section of
the "Status" tab

ghwood committed:
Merge pull request #1 from ghwood/ghwood-patch-1

Clarify "Replace" in I-D submission form

jennifer-richards commented on ietf/submit/forms.py:
```diff
@@ -502,7 +502,7 @@ def clean_name(self):
 
 class ReplacesForm(forms.Form):
-    replaces = SearchableDocumentsField(required=False, help_text="Any Internet-Drafts that this document replaces (approval required for replacing an Internet-Draft you are not the author of)")
+    replaces = SearchableDocumentsField(required=False, help_text="An I-D submission will be automatically attached to the previous version of the I-D with the same name, or assigned "-00" version if no previous version exists. If this submission is intended to replace an I-D with a different name (e.g. when an individual I-D is adopted as a Working Group document), please enter that name here. ( ...
```
There's a syntax error here because of the `"-00"` inside the quoted string. You can change the outer quotes to `'` to fix this.

jennifer-richards reviewed (changes requested):
See inline

NGPixel committed:
refactor: update node, eslint, neostandard + fix esm (#8083)

* chore: update dependencies

* fix: eslint + neostandard

* fix: add corepack prompt env var to init script

* docs: Update README.md

---------

Co-authored-by: Robert Sparks <rjsparks@nostrum.com>

NGPixel committed:
ci: update base image target version to 20241114T1703

NGPixel committed:
ci: fix tests.yml workflow

ghwood committed:
Merge branch 'ietf-tools:main' into main

ghwood committed:
fix: clarify "Replace" in I-D submission form #8205

update quote syntax issue on previous update help text In the "Replacement Information" section of the "Status" tab to fix #8059

ghwood committed:
Merge pull request #2 from ghwood/ghwood-patch-1

fix: clarify "Replace" in I-D submission form #8205

rjsparks commented:
Thanks for pushing on this. Some thoughts below:

> An I-D submission will be automatically attached to the previous version of the I-D with the same name, or assigned "-00" version if no previous version exists. If this submission is intended to replace an I-D with a different name (e.g. when an individual I-D is adopted as a Working Group document), please enter that name here. (Approval required to replace an I-D for which you are not an author.)

The first sentence doesn't work for me - it sees to imply "-00" is a thing that happens at this point, but -00 was chosen by the submitter long before the submitter gets to this form.

I also don't think "automatically attached to a previous" is what happens, or is the right model of what's happening for other people to build. Instead, we need to reinforce that there are versions of an Internet-Draft, and that an Internet-Draft might get replaced with some other Internet-Draft(s).

Maybe imperative statements rather than explanation would work better? My alternate suggestion:

-----

Do not enter anything here if you are only submitting the next version of your Internet-Draft. Only enter items here if this submission is intended to replace an I-D with a different name. A typical use of this field is to note what individual I-Ds are replaced by a new -00 Working group I-D. Note that additional approval will be required to replace an I-D for which you are not an author.

rjsparks commented:
mph - just spotted that you're merging back into your fork's main. That will be painful for you in the long run. I'll followup with you on it offline.

For anyone else that may stumble across this - it's better to leave main in your fork alone - sync it with ietf-tools/main periodically. Build your PRs in another branch.

rjsparks committed:
Revert "refactor: update node, eslint, neostandard + fix esm (#8083)"

This reverts commit 649879efd745470f6e0cc6768d889f45640e1505.

rjsparks committed:
Revert "ci: update base image target version to 20241114T1703"

This reverts commit f11144017ed788bc7a38a1d028127434f0d50eb4.

rjsparks committed:
Revert "ci: fix tests.yml workflow"

This reverts commit 39231321c49291565a39608b98740e098c74dda7.
